kenkono/e-learning-laravel
# ELS-TASK-13-activeStatus

## Description / Purpose (checkpoints)
- Show activity.

## How To Test This PR
After answering question, go to 'localhost:8000/home'
You can see the status that already taken or not.
 
## **Commands**
Commands needed
`php artisan migrate` 
Insert the test data
`php artisan migrate:refresh --seed`
